js/reference.js

```diff
@@ -1807,18 +1808,21 @@
         },
 
         /**
-         * Deletes the referenced resources.
+         * Deletes the referenced resource's tuples.
          * NOTE This will ignore the provided sort and paging on the reference, make
          * sure you are calling this on specific set or rows (filtered).
          *
-         * @param {Object} contextHeaderParams the object that we want to log.
+         * @param {Object[]} tuples optional array of objects with just the data needed to construct the uri, if not-defined use the reference ermrestCompactUri (should only be empty for association reference)
```

Comment:
I don't think it's optional anymore. you might want to remove the association reference part too.